trinity/chains/__init__.py

```diff
@@ -56,6 +65,50 @@
 )
 
 
+def validate_genesis(chain_config: ChainConfig) -> (BlockHeader, int):
```

Comment:
Can we change this to `validate_EIPXXX_genesis` (with the `XXX` replaced by the EIP number), as well as maybe linking to the EIP in the docstring for easy reference.